gcp_variant_transforms/testing/integration/run_tests.py

```diff
@@ -188,9 +189,15 @@ def run_assertion(self):
 class QueryFormatter(object):
   """Formats a query.
 
-  Replaces keyword TABLE_NAME and eventually macros in the query.
+  Replaces macros and variable TABLE_NAME in the query.
   """
 
+  class _QueryMacros(enum.Enum):
+    NUM_ROWS_QUERY = 'SELECT COUNT(0) AS num_rows FROM {TABLE_NAME}'
+    SUM_START_QUERY = ('SELECT SUM(start_position) AS sum_start FROM {'
```

Comment:
nit: consider the following format as it's more readable (in general, try to avoid breaking lines as much as possible).
```
   SUM_START_QUERY = (
       'SELECT SUM(start_position) AS sum_start FROM {TABLE_NAME}')
```

Reply:
Thanks, done.